### ISSUE TITLE
[FEAT] Rename and surface `sraRunAccession`

### ISSUE THREAD

**Issue by emmahodcroft** [pathoplexus/pathoplexus#331]
To make it easier for users to link raw-read data with records submitted directly to Pathoplexus, we propose:

- Renaming the current `sraRunAccession` field to `insdcRawReadsAccession` in order to make this more generic (as raw-reads can be submitted in place other than SRA)
- 'Surfacing' this field in our docs and templates so that users know they can add this manually*

(*currently the `sraRunAccession` field is used by the INSDC-ingest to add any available raw-read accessions)

@chaoran-chen has tested adding this field as currently named to uploaded data (in `demo`) and there seems to be no issue with users specifying this field - it comes through as expected - so no functionality change on that side is expected to be needed.

Steps needed:
- [ ] Rename `sraRunAccession` to `insdcRawReadsAccession` in all code; ensure this does not impact mapping this field while doing ingest from NCBI Virus
- [x] #336
- [ ] Update docs & website
  - [ ] [Metadata format page](https://pathoplexus.org/docs/concepts/metadataformat)
  - [ ] [Downloadable Metadata description file](https://github.com/pathoplexus/pathoplexus/blob/main/monorepo/website/public/metadata_descriptions.csv)
  - [ ] Submission template on the Submit pages (can't find a good link, sorry)

**Comment by anna-parker** [comment]
@chaoran-chen did you check that there was no version bump of fields ingested from INSDC with this renaming?

**Comment by anna-parker** [comment]
I'm personally ok having this version bump, but an alternative would just be to modify the displayName.

**Comment by chaoran-chen** [comment]
@anna-parker, thanks for bringing this up! I wasn't aware of this issue and didn't test anything w.r.t. the renaming but only that `sraRunAccession` is a field that the user can use at the moment. In general, I think that it makes sense to fully rename it and personally wouldn't mind the version bump. What do others think, @emmahodcroft, @theosanderson?

**Comment by theosanderson** [comment]
We discussed and I think went for the versionBump

**Comment by anna-parker** [comment]
Hey I was just looking into https://github.com/pathoplexus/pathoplexus/issues/331 and I created: https://github.com/pathoplexus/pathoplexus/issues/336 - when submitting raw reads to ENA they require a study and a bioproject in order to submit raw reads. From our previous experience with assemblies I have a strong assumption this cannot be altered. 

If users first submit raw reads to INSDC and then the consensus to us we would really need to know the biosample and the bioproject accession (we can look this up) so that we can upload the consensus sequence so that the two are connected (as we now know that we cannot change associated bioprojects).

Or we could let users first submit the consensus sequence to us and then upload the raw reads using the biosample and bioproject accession that we created. 

Either way we need to ensure that the biosample is not created twice and that both assembly and raw read (experiment in ENA) are linked to the same bioproject. 

Probably we should we chose one of these approaches to not complicate matters.

**Comment by anna-parker** [comment]
Update: we can link an assembly to a raw read - so asking users to first submit the raw read and then have us upload the consensus sequence using the bioproject and biosample they created might be the better approach.

**Comment by theosanderson** [comment]
I don't think we can really impose that much in the way of constraints on users here - they may need to submit a consensus sequence before raw reads - for example in an urgent situation

**Comment by anna-parker** [comment]
Could we add a warning if users fill out the sraAccession and add docs on how to upload raw reads to make sure that raw reads and consensus sequences are both associated to the same project?

**Comment by theosanderson** [comment]
> Could we add a warning if users fill out the sraAccession 

If users fill out SRA accession and submit openly then we could (technically) add the consensus to the same project - right?

**Comment by anna-parker** [comment]
Yes - we can submit to the same bioproject as the SRA accession - I just think we should make this clear to the submitters in the docs as this is confusing.

**Comment by emmahodcroft** [comment]
Agree that I think we probably can't enforce strict order constraints here long-term. (Short-term I think it's fine if while we're finalizing we ask Vitor to go for a specific order if that's easier for us for now.) 

Am I understanding that the main thing is that in either case, we need to know the existing bioproject and biosample in order to link them properly? 

So:
1. They upload consensus to us. We give them bioproject and biosample and must use these when uploading to SRA. Then they edit their Pathoplexus samples to link the raw read IDs?
2. They upload to SRA first. Then they upload consensus to us and specify bioproject and biosample, and raw-read-ids, so that we can upload the consensus to the same 'place', and include the raw-read-ids in the Pathoplexus info?

Or am I missing something still?
